packages/shared/src/lib/dateFormat.ts

```diff
@@ -22,11 +29,12 @@ export function postDateFormat(
 
   if (isSameDay(date, subDays(now, 1))) return 'Yesterday';
 
-  return date.toLocaleString('en-US', {
+  const options: Intl.DateTimeFormatOptions = {
     month: 'short',
     day: '2-digit',
-    year: 'numeric',
-  });
+  };
+  if (!isSameYear(date, now)) options.year = 'numeric';
+  return date.toLocaleString('en-US', options);
```

Comment:
Maybe just double check product and design are ok with this :)

Reply:
This is actually according to the design

<img width="326" alt="SCR-20230206-vlq" src="https://user-images.githubusercontent.com/1488164/217069674-a739a6ae-2136-46ef-bf27-e3ef861ed65c.png">